docs/source/resources/01a_object_detection.rst

```diff
@@ -123,6 +123,18 @@ the models. All values are reported in percentage.
 
 All images from the 80 object categories in the MS COCO (val 2017) dataset were processed.
 
+Test Conditions
+^^^^^^^^^^^^^^^
+
+The following test conditions were followed:
+ | - The tests were performed using `pycocotools <https://pypi.org/project/pycocotools/>`__ on the
+     MS COCO dataset
+ | - The score threshold is based on PeekingDuck's default settings of ``0.2``, ``0.3``, and ``0.25`` 
+     for YOLO, EfficientDet, and YOLOX respectively
+ | - For YOLOX, the class agnostic NMS has been set to ``False``
+ | - The evaluation metrics have been compared with the original repository of the respective object
+     detection models for consistency
+
```

Comment:
Sorry, I meant the configs such as thresholds and NMS should be written in the internal spreadsheet, not in the docs.